### DISCUSSION TITLE
What do you think about transitioning to Linaria ?

### DISCUSSION THREAD

**Discussion by dukesx** [mantinedev/mantine#3579] [Feedback]
## Why 
It seems that React is hooked up on the idea of Server Components, so naturally we can say that they aren't going anywhere, anytime soon. 
So, Based on ZERO runtime and a Next JS 13 ready plugin, i think its high time Mantine thinks of migrating from Emotion to Linaria, since unlike Linaria and Styled Components  Emotion is reaching nowhere and soon all react based frameworks are bound to include server components at some point, so if you are using Gatsby or Remix and not worried about this transition ? Just... hold on to that thought. 

### How
I think the syntax is usually the problem, when it comes to migration, where Linaria provides syntax support for both "styled" and "css" props. It is NOT GOING TO BE EASY but for now lets just discuss where Mantine heads to next and dig out possibilities of how we can make Dev experience better for everyone.

**Comment by rtivital** [comment]
There is no point in migrating to other css-in-js library, especially Linaria which is not very popular. If it will be impossible to use emotion with future versions of meta frameworks, I will migrate Mantine to native CSS.
